tools/floodgate/index.html

```diff
@@ -29,6 +29,10 @@ <h2 class="project-name">
         <div id="promoteFiles" class="hidden">
           <button type="button">Promote Files</button>
         </div>
+        <div id="updateFragments" class="hidden">
+          <button type="button">Update Fragments</button>
```

Comment:
Nit: outside a form, the `type="button"` attribute is redundant, it can probably be removed for the buttons listed above.